CRFModuleValidation/Cardio_12MT_ORKTask.json

```diff
@@ -0,0 +1,79 @@
+{
+    "taskIdentifier"            : "Cardio 12MT",
+    "schemaIdentifier"          : "Cardio 12MT",
+    "taskType"                  : "cardio",
+    "taskOptions"               : { "walkDuration" : 720.0, "restDuration" : 60.0 },
+    "localizedSteps"            : [
+                                   {
+                                   "identifier"   : "heartRisk",
+                                   "text"         : "Do not attempt this test if you have experienced unstable angina, a myocardial infarction (heart attack) during the previous month, need supplemental oxygen to walk, or if you feel that walking for a mile will be a challenge for you."
+                                   },
+                                   {
+                                   "identifier"   : "workoutInstruction1",
+                                   "title"        : "Running Instructions",
+                                   "text"         : "For this test you will need a flat and uncrowded area outdoors where you can run for 12 minutes at a comfortable pace without interruption. Your goal is to run as fast as you can in 12 minutes."
```

Comment:
"run as fast as you can _for_ 12 minutes" or "run as _far_ as you can in 12 minutes"